extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
+// By: BOP <https://scratch.mit.edu/users/CEFSharp/>
+
+(function (Scratch) {
+  "use strict";
+
+  let chat = null;
+  let token = null;
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
+        name: "Twitch",
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
+        docsURI:
+          "https://github.com/Tekinical/tekinical.github.io/blob/main/Twitch%20Extension%20Doc.md",
```

Comment:
The docs must be written in our version of markdown and stored in the docs folder. Otherwise, delete this.

Reply:
Last task.

Reply:
Finished.